babel.config.js

```diff
@@ -8,7 +8,8 @@ module.exports = function (api) {
   ];
 
   const plugins = [
-    '@babel/plugin-proposal-optional-chaining'
+    '@babel/plugin-proposal-optional-chaining',
+    '@babel/plugin-transform-runtime'
```

Comment:
This was to prevent Babel throwing an error -  `regeneratorRuntime is not defined`.

Fix was found here https://github.com/babel/babel/issues/5085#issuecomment-277544677